packages/workbox-broadcast-update/src/BroadcastCacheUpdate.ts

```diff
@@ -24,11 +24,11 @@ const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
 
 
 // Give TypeScript the correct global.
-declare var self: ServiceWorkerGlobalScope;
+declare let self: ServiceWorkerGlobalScope;
 
 export interface BroadcastCacheUpdateOptions {
   headersToCheck?: string[];
-  generatePayload?: (options: CacheDidUpdateCallbackParam) => Object;
+  generatePayload?: (options: CacheDidUpdateCallbackParam) => Record<string, any>;
```

Comment:
I _think_ this is a safe change (it was done automatically via `--fix`), but I guess there's a chance it might be considered a breaking change?

Reply:
Seems safe to me? I guess the worse case scenario is someone has to update their typings. I'm not sure if that's considered breaking or not.